api/config/cdr_config_prod.json

```diff
@@ -7,8 +7,17 @@
       "servicePerimeter": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_prod",
       "authDomainName": "all-of-us-registered-prod",
       "authDomainGroupEmail": "all-of-us-registered-prod@firecloud.org",
-      "datasetsBucket": "gs://fc-aou-datasets-registered/",
```

Comment:
why remove these?  I don't have much context here.

Reply:
This bucket was never actually used or, perhaps not even created. Currently, we don't have any GCS assets that are part of the CDR in registered tier.

Reply:
As to why it was here: originally the intent was to create a bucket in every perimeter, but we decided not to create extraneous buckets in the end. This was left over from that.